client/me/purchases/remove-purchase/index.jsx

```diff
@@ -341,7 +341,7 @@ class RemovePurchase extends Component {
 			);
 		}
 
-		if ( this.props.isAtomicSite ) {
+		if ( this.props.isAtomicSite && ! purchase.productName.includes( 'Search' ) ) {
```

Comment:
I feel like this should use a function that checks against the full list of products rather than being dependent on the actual text in the name. Main concern is if there are products such as "Search Engine Optimization". Do we have an isJetpackSearchProduct() call or something?

Reply:
Great point. I re-wrote the condition using `isJetpackSearch` utility function.